modules/openapi-generator/src/main/resources/Java/libraries/rest-assured/ApiClient.mustache

```diff
@@ -33,8 +33,8 @@ public class ApiClient {
 
      {{#apiInfo}}
      {{#apis}}
-    public {{classname}} {{classVarName}}() {
-        return {{classname}}.{{classVarName}}(config.baseReqSpec.get());
+    public {{classname}} {{classVarName}}Api() {
+        return {{classname}}.{{classVarName}}Api(config.baseReqSpec.get());
```

Comment:
@spacether We will need to flag this change as a breaking change without fallback

cc @viclovsky any concern with this change and the change in the line below: https://github.com/OpenAPITools/openapi-generator/pull/1883/files#diff-06d453e322d6a8c6104519bc6df24fb1R40 ?

Reply:
Suggestion: undo the changes in the rest-assured templates so that we can merge this PR and make those breaking changes in another PR instead (from the title of this PR, it's hard to tell the API class name been renamed)

Reply:
Removed the rest-assured fix for issue #13 per your suggestion

Reply:
> Removed the rest-assured fix for issue #13 per your suggestion

Thanks. It would be great to keep backward compatibility. I suggested how to fix this issue here https://github.com/OpenAPITools/openapi-generator/issues/13#issuecomment-390635959